pkg/testcase/service.go

```diff
@@ -1,101 +1,73 @@
 package testcase
 
 import (
-	"fmt"
-
 	"github.com/rancher/distros-test-framework/pkg/assert"
-	"github.com/rancher/distros-test-framework/pkg/customflag"
 	"github.com/rancher/distros-test-framework/shared"
 
-	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-func TestServiceClusterIp(deployWorkload bool) {
-	if deployWorkload {
-		fmt.Println("ARCH", customflag.ServiceFlag.ClusterConfig.Arch.String())
-		_, err := shared.ManageWorkload(
-			"create",
-			"clusterip.yaml",
-			customflag.ServiceFlag.ClusterConfig.Arch.String(),
-		)
-		Expect(err).NotTo(HaveOccurred(),
-			"Cluster IP manifest not deployed")
-	}
+func TestServiceClusterIP(deleteWorkload bool) {		
+	_, err := shared.ManageWorkload("apply", "clusterip.yaml")
```

Comment:
Apply could overwrite -- is this definitely what we want for ALL test cases that use this function? 

Also, do we ALWAYS want to apply with this? Are there any cases that call this function that are expecting the resource to not be there and attempt a delete?


(same questions for all similar changes in this file)

Reply:
As long as the deployment file is unchanged, it should be fine, right? This is one of the reason I did not want to have one method to handle the operations as it makes the Test more abstract

We can definitely add a check for the resource to exist before performing deletion but atm it does not exist

Reply:
Yeah abstract tests aren't great and there is definitely room for improvement overall in that regard. You're right in that if the file doesn't change it's fine, but I'm wondering more for the case of "Let's try to delete something that we never applied in the first place" -- in that case we'd have to use a different method altogether I guess, which is fine!

I think it's okay as is 👍